jujugui/static/gui/src/app/components/entity-content/entity-content.js

```diff
@@ -70,27 +115,50 @@ YUI.add('entity-content', function() {
     */
     _generateDescription: function(entityModel) {
       if (entityModel.get('entityType') === 'charm') {
-        return <div className="entity-content__description">
-            <h2>Description</h2>
-            <p>{entityModel.get('description')}</p>
-          </div>;
+        // Have to convert {0: 'database'} to ['database'].
+        var tags = [],
+            entityTags = entityModel.get('tags'),
+            index;
+        for (index in entityTags) {
+          tags.push(entityTags[index]);
+        }
+        return (
+          <div className="row entity-content__description">
+            <div className="inner-wrapper">
+              <div className="twelve-col">
+                <p>{entityModel.get('description')}</p>
+              </div>
+              <div className="four-col entity-content__metadata last-col">
+                <h4>Tags</h4>
+                <ul>
+                  {this._generateList(tags, this._handleTagClick)}
+                </ul>
+              </div>
+            </div>
+          </div>
+        );
       }
     },
 
     render: function() {
       var entityModel = this.props.entityModel;
       return (
-        <div className="row entity-content">
-          <div className="inner-wrapper">
-            <main className="seven-col append-one">
-              {this._generateDescription(entityModel)}
-              <juju.components.EntityContentReadme
-                entityModel={entityModel}
-                renderMarkdown={this.props.renderMarkdown}
-                getFile={this.props.getFile} />
-              {this._generateOptionsList(entityModel)}
-            </main>
+        <div className="entity-content">
+          {this._generateDescription(entityModel)}
+          <div className="row">
+            <div className="inner-wrapper">
+              <div className="seven-col append-one">
+                <juju.components.EntityContentReadme
+                  entityModel={entityModel}
+                  renderMarkdown={this.props.renderMarkdown}
+                  getFile={this.props.getFile} />
+              </div>
+              <div className="four-col">
+                <p>Placeholder for project information.</p>
```

Comment:
Maybe we shouldn't commit dummy stuff like this, just in case we don't get to it before release.